api/src/wfl/main.clj

```diff
@@ -41,9 +41,7 @@
   "Map command names to their run functions."
   (letfn [(varify [ns] (require ns) (ns-resolve ns 'run))
           (namify [ns] (last (str/split (name ns) #"\.")))]
-    (let [namespaces '[wfl.dx
-                       wfl.metadata
-                       wfl.server]]
+    (let [namespaces '[wfl.server]]
       (assoc (zipmap (map namify namespaces) (map varify namespaces))
              "help"         #'help
              "version"      #'version
```

Comment:
Also left over from command-line days.

Reply:
Should it be removed too then?

Reply:
If you feel like it. I believe the PR is OK to merge. This can wait.